lib/vix/vips/image.ex

```diff
@@ -42,6 +42,31 @@ defmodule Vix.Vips.Image do
   @impl Type
   def to_erl_term(ref), do: %Image{ref: ref}
 
+  # Implements the Access beahviour for Vix.Vips.Image to allow
```

Comment:
Let's add a bit about this to `moduledoc` for the better discoverability?

Reply:
Done, additional commit pushed.